docs/extension-system/_index.md

```diff
@@ -0,0 +1,148 @@
+---
+title: 'Extension system'
+date: 2024-01-23T00:00:00+00:00
+weight: 60
+geekdocRepo: https://github.com/owncloud/web
+geekdocEditPath: edit/master/docs/extension-system
+geekdocFilePath: _index.md
+geekdocCollapseSection: true
+---
+
+{{< toc >}}
+
+## Concepts and Building Blocks
+
+The ownCloud Web can be extended through various entry points with custom **apps** and **extensions**.
+
+### Distinction between Apps and Extensions
+
+An Application in the context of ownCloud Web is an artifact which can be distributed to and installed by an ownCloud admin.
+It serves two main purposes:
+1. It makes the full app viewport (everything below the top bar) available to the application developer for any custom
+   application code. This includes the ability to define views with routes, navigation items for the left side bar, and more.
+2. Through the `extensions` key in the application interface the developer can register extensions of any extension type.
+   Those extensions will become available in standardized extension points and for being queried from the extension registry
```

Comment:
... and will be queried?

I assume you mean they will become available for being queried - but that doesn't work well with the two parts of the sentence imho

Reply:
Changed the paragraph to `Through the `extensions` key in the application interface you can register extensions of any extension type. Those extensions 
   are then available in standardized extension points. Additionally, they can be queried from the extension registry for 
   your own purposes.`. How about that?